README.md

```diff
@@ -2,7 +2,7 @@
 
 This README is under construction as we work to build a new community driven high performance key-value store.
 
-This project was forked from the open source Redis project right before the transition to their new source available licenses.
+This project was forked from the open source Valkey project right before the transition to their new source available licenses.
```

Comment:
All of the references in this file should stay the same, since they are explicit references to the old Redis project.

Reply:
reverted